core/metrics-core-service/src/main/java/org/hawkular/metrics/core/service/MetricsServiceImpl.java

```diff
@@ -940,11 +940,7 @@ public void setDefaultTTL(int defaultTTL) {
 
     @Override
     public Observable<Boolean> idExists(final MetricId<?> metricId) {
```

Comment:
My IDE can't actually even find this being used anywhere. Any idea why we have it?